testsuite/features/step_definitions/common_steps.rb

```diff
@@ -936,3 +936,8 @@ def server_secret
   count_failures = failings.length
   raise "\nFailures in event history found:\n\n#{failings}" if count_failures.nonzero?
 end
+
+And(/I should see a list item with text "([^"]*)" and bullet with class "([^"]*)"/) do |text, className|
```

Comment:
I think we should rephrase the sentence, is fine to have as a parameter the text in the item list, but I think is better to don't ask for a class name in our steps, it's a bit less human readable and near to coding.